tensorflow_similarity/samplers/tfrecords_samplers.py

```diff
@@ -90,7 +90,13 @@ def TFRecordDatasetSampler(
     Returns:
         A `TF.data.dataset` ready to be consumed by the model.
     """
-    shards_list = [str(i) for i in Path(shard_path).glob(shard_suffix)]
+    shards_list = [
+        i.decode()
+        for i in tf.io.matching_files(os.path.join(shard_path, shard_suffix))
+        .numpy()
+        .tolist()
+    ]
+    print(f"found {shards_list}")
```

Comment:
Hi @owenvallis,
Thank you for your reply. 

What do you think about using 
```
from absl import logging
...
logging.info(f"found {shards_list}")
```
instead of the `print` statement. Then we can save an additional flag and the user can still control the verbosity of the logging by setting the level?

Reply:
Is there a reason, the package is using `print` over `logging`  statements?

Reply:
Hi @hanneshapke,

Thanks for the PR by the way, this will be a nice update. 

Regarding logging, I agree and think we should likely move the package to using logging in the near future. We have been using Print as much of the output is meant to be seen in notebooks, but I think moving to absl logging could make a lot of sense. We likely want to do this for the whole package, so I'll open an issue for us to revisit this.

I think it would be fine to add the logging for this line but maybe we should set the level for this to DEBUG. If we migrate our Print output to logging.INFO, then we'll likely still only want to see the full list of shards when debugging.

wdyt?

Reply:
@owenvallis Sounds great. Just pushed the little change from `info` to `debug`.